docker/python/start-web.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Wait for PostgreSQL
+python manage.py wait_for_db
+
+# Run migrations
+python manage.py migrate
+
+# Check environment and start the appropriate server
+if [[ "$DJANGO_ENV" = "PRODUCTION" ]]; then
+  gunicorn -b 0.0.0.0:8000 \
+    --log-level ${DJANGO_LOG_LEVEL,,} \
+    --access-logfile - Forum.wsgi:application
```

Comment:
Maybe we should specify automatic detection of the number of workers based on the number of processor cores?


    --workers $(echo "$WEB_CONCURRENCY" | sed 's/[^0-9]//g') \